types/angular/index.d.ts

```diff
@@ -1,6 +1,9 @@
 // Type definitions for Angular JS 1.6
 // Project: http://angularjs.org
-// Definitions by: Diego Vilar <http://github.com/diegovilar>, Georgii Dolzhykov <http://github.com/thorn0>, Caleb St-Denis <https://github.com/calebstdenis>
+// Definitions by: Diego Vilar <http://github.com/diegovilar>
+//                 Georgii Dolzhykov <http://github.com/thorn0>
+//                 Caleb St-Denis <https://github.com/calebstdenis>
+//                 Eduardo Garcia <https://github.com/thewarpaint>
```

Comment:
These things are parsed by the bot to notify the reviewers, so this is essentially a request to become a reviewer. 1) Did you mean it? 2) You changed the format of the list. Make sure the bot understands it. I have doubts about it.